tool/src/js/conversions.rs

```diff
@@ -337,8 +337,8 @@ pub fn gen_value_rust_to_js<W: fmt::Write>(
             gen_rust_reference_to_js(underlying.as_ref(), in_path, value_expr, "null", env, out)?;
         }
         ast::TypeName::Writeable => todo!(),
-        ast::TypeName::StrReference => todo!(),
-        ast::TypeName::PrimitiveSlice(_) => todo!(),
+        ast::TypeName::StrReference(_) => todo!(),
```

Comment:
thought: this is what i was worried about, but turns out this is already todo :)